modules/recast/SCsub

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+Import('env')
+
+# Not building in a separate env as core needs it
+
+# Thirdparty source files
+if (env['builtin_recast'] != 'no'):
+    thirdparty_dir = "#thirdparty/recastnavigation/Recast/"
```

Comment:
The "Recast/" folder seems redundant.

Reply:
Yes, as @Causeless mentions, this is the reason the folder is structured like this.